README.md

```diff
@@ -1,6 +1,5 @@
-[![Build Status](https://travis-ci.org/mongodb/mongo-swift-driver.svg?branch=master)](https://travis-ci.org/mongodb/mongo-swift-driver)
```

Comment:
Since we'll be opting to get rid of travis soon, I just decided to drop the badge altogether instead of updating it.